packages/react-components/src/components/CameraButton.tsx

```diff
@@ -139,12 +148,15 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
   const [waitForCamera, setWaitForCamera] = useState(false);
   const localeStrings = useLocale().strings.cameraButton;
   const strings = { ...localeStrings, ...props.strings };
-  const onRenderCameraOnIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOn" />
-  );
-  const onRenderCameraOffIcon = (): JSX.Element => (
-    <HighContrastAwareIcon disabled={props.disabled || waitForCamera} iconName="ControlButtonCameraOff" />
-  );
+  const [announcerString, setAnnouncerString] = useState<string>(strings.cameraActionTurnedOffAnnouncment);
```

Comment:
(From Teams chat) Can we avoid the set state and have this be
```ts
const announcerString = (props?.checked && !waitForCamera) ? strings.cameraActionTurnedOnAnnouncment : strings.cameraActionTurnedOffAnnouncment
```
Although in writing this again I see it may be called on first render which we won't want

Reply:
Having it inside the `onRenderCameraOnIcon` though isn't ideal as if Contoso supplies their own icon they lose this feature

Reply:
Maybe we keep the setState but put it in `onToggleClick`?

Reply:
I will move it there and implement for the microphone button as well.